server_environment/__manifest__.py

```diff
@@ -20,8 +20,11 @@
 
 {
     "name": "server configuration environment files",
-    "version": "11.0.1.0.1",
-    "depends": ["base"],
+    "version": "11.0.1.2.0",
```

Comment:
I would put it as a major version. While it doesn't break compatibility, the paradigm is reversed as it sort of deprecate `server_environment_files`

Reply:
done